implementations/rust/ockam/ockam_core/src/message.rs

```diff
@@ -72,6 +72,7 @@ pub trait Message: Encodable + Decodable + Send + 'static {}
 impl Message for () {}
```

Comment:
Off-topic: should we remove those implementations to discourage users from sending such basic types to workers? I think it's better to wrap even such simple data into dedicated structs. Even though we don't use that approach in our examples

Reply:
We should either do that or implement it for all serializable/deserializable stdlib types.

Reply:
Thinking a bit, of these, I think I'm in favor of implementing it for everything, tbh. More convenient and less of a roadblock than it would be to tell people "Hey, you need to define a newtype wrapper here". It will keep both our examples simple, as well as requiring less work for users.

Reply:
Yea I'd second that. At times I've wanted to just send an `i64` in a demo which used to work and now doesn't.
In reality primitive type messages are probably less useful outside of a quick prototype but if that's all the user wants they shouldn't have to wrap the type in a struct that does nothing else